src/NServiceBus.RavenDB/Internal/DocumentStoreInitializer.cs

```diff
@@ -111,6 +112,12 @@ static void EnsureClusterConfiguration(IDocumentStore store)
 
                 var topology = getTopologyCmd.Result.Topology;
 
+                // Currently do not support clusters
+                if (topology.AllNodes.Count != 1)
+                {
+                    logger.Error("RavenDB Persistence does not support RavenDB clusters. Only single node setup is supported.");
+                }
+
                 // Currently do not support clusters with more than one possible primary member. Watchers (passive replication targets) are OK.
                 if (topology.Members.Count != 1)
```

Comment:
What's the point of keeping this then?

Reply:
None, except I guess that not throwing anymore an exception, strictly speaking, is a breaking change. This PR is against 6.5.0 and thus we can only log an error statement. Thoughts?

Reply:
Oh nevermind, I thought I was looking at the master version.